Orange/widgets/data/owtranspose.py

```diff
@@ -101,6 +99,8 @@ def send_report(self):
 
     app = QApplication([])
     ow = OWTranspose()
+    d = Table("iris")
```

Comment:
This is the case where it doesn't work. (To see this, add `print(self.feature_names_column)` after `self.openContext()`.)